exercises/all-your-base/example.rs

```diff
@@ -73,5 +78,9 @@ pub fn convert<P: AsRef<[Digit]>>(
     }
     // fix order of digits
     res.reverse();
-    Ok(res)
+    if res.is_empty() {
+        Ok(vec![0])
+    } else {
+        Ok(res)
+    }
```

Comment:
This can be simplified to avoid an allocation:

```suggestion
    if res.is_empty() {
        res.push(0);
    } 
    Ok(res)
```